src/emmy/calculus/derivative.cljc

```diff
@@ -292,7 +293,7 @@
    (let [entry (get-in structure path)]
      (deep-partial f structure path entry)))
   ([f structure path entry]
-   (if (v/numerical? entry)
+   (if (v/scalar? entry)
```

Comment:
@littleredcomputer this change leaves us free to make `numerical?` return `false` for our differential AND tape instances.